docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  locutus:
+    build:
+      context: ..
+      dockerfile: ./docker/Dockerfile
+    image: locutus:docker
+    ports:
+      - target: 50509
```

Comment:
so we assume we are using the default port then? maybe since you just made this parametrizable we should use an other one, in case the user for some weird reason happens to run the node both in a container and in its host so there is no collision.

Reply:
Agreed.  Also, someone using this docker-compose could customize it to their needs.  I made the exposed port 50599 to avoid accidental collision.